### PR TITLE
fix #1484 Add Loggers.useVerboseConsoleLoggers, default to no TRACE/DEBUG

### LINKED ISSUE #1484: Loggers defaults to very noisy if SLF4J is not present
Reactor "console logging" is noisy - you can't limit the levels at all, so all TRACE and DEBUG level debugs come out on the console. If that really is unavoidable then IMO it should not be the default (when SLF4J is not available). The JDK logger isn't perfect, but at least it is quieter by default.

This problem is particularly acute when using reactor-netty because netty is noisy and  reactor-netty even more so - it emits stacktraces for TRACE logging, even when there is no error. The DEBUG level is also rather noisy (logs all request-response).

The workaround, in case anyone else needs it: `reactor.logging.fallback=JDK`.

### PR DESCRIPTION
**Pull request by simonbasle**
`Behavior change`: `useConsoleLoggers()` does not TRACE or DEBUG anymore.

### CODE REVIEW

File: reactor-core/src/test/java/reactor/util/ConsoleLoggerTest.java [hunk 1]
```diff
@@ -88,6 +88,26 @@ public void traceNulls() {
 				.contains("param null is null");
 	}
 
+	@Test
+	public void traceDismissedInNonVerboseMode() {
+		PrintStream out = System.out;
+		Logger log = new Loggers.ConsoleLogger("test", false);
+		System.setOut(new PrintStream(outContent));
```
Comment (bsideup) [inline reactor-core/src/test/java/reactor/util/ConsoleLoggerTest.java]: `errContent` is not set but asserted

Comment (simonbasle) [reply reactor-core/src/test/java/reactor/util/ConsoleLoggerTest.java]: indeed... actually setting the System printstreams is not even necessary and doesn't help the test, so I simplified it

**Comment by simonbasle** [comment]
(rebased on current master)

**Comment by simonbasle** [comment]
@smaldini any opinion about the behavioural aspect of the change?

**Pull request merged**